src/Control/Lens/Internal/Coerce.hs

```diff
@@ -16,9 +10,7 @@
 -- Stability   :  experimental
 -- Portability :  non-portable
 --
--- This module provides a shim around 'coerce' that defaults to 'unsafeCoerce'
--- on GHC < 7.8. It also exposes a type-restricted version of '(#.)' that
--- works around a bizarre GHC 7.10–specific bug.
+-- This module provided a shim around 'coerce'.
```

Comment:
s/provided/provides/

Also, it would be worthwhile to say _why_ this shim exists. Previously, it was due to the need to use `unsafeCoerce`, but now it's due to the need to work around a GHC 7.8-specific `Coercible` solver bug. Say that.